packages/clerk-js/src/ui/components/OrganizationSwitcher/OrganizationSwitcherPopover.tsx

```diff
@@ -133,9 +133,14 @@ export const OrganizationSwitcherPopover = React.forwardRef<HTMLDivElement, Orga
         <PopoverCard.Root
           elementDescriptor={descriptors.organizationSwitcherPopoverCard}
           ref={ref}
+          role='dialog'
+          aria-label='organization switcher popover'
           {...rest}
         >
-          <PopoverCard.Main elementDescriptor={descriptors.organizationSwitcherPopoverMain}>
+          <PopoverCard.Main
+            elementDescriptor={descriptors.organizationSwitcherPopoverMain}
+            role='menu'
```

Comment:
This would only make sense if we'd use the rest of the menu syntax (https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/menu_role) which I think we don't. So in current state this should be dropped then

Reply:
Nested components use the role of `menuitem`, do we need something more ?